models/demos/deepseek_v3/conftest.py

```diff
@@ -92,3 +92,8 @@ def ccl(mesh_device):
     This is used to test distributed operations in DeepSeek modules.
     """
     return CCL1D(mesh_device)
+
+
+@pytest.fixture(scope="session")
+def deepseek_cache_path():
```

Comment:
let's add this in a separate PR maybe, I thought that for now we wanted to keep things cacheless